packages/block-editor/src/components/default-block-appender/style.native.scss

```diff
@@ -1,7 +1,8 @@
 // @format
 
 .blockHolder {
-	flex: 1 1 auto;
+	padding-left: 16;
+	padding-right: 16;
```

Comment:
Just curious, why aren't we using `px` here?

Reply:
This should be fixed now.